src/relayclient/RelayClient.ts

```diff
@@ -360,3 +360,24 @@ export class RelayClient {
     return forwarderAddress
   }
 }
+
+export function _dumpRelayingResult (relayingResult: RelayingResult): string {
```

Comment:
`RelayClient.ts` is a weird placement for this function. Why?

Reply:
RelayProvider is weirder place for a method parse a return value of RelayClient.

I really wished this formatter to be a method off the RelayingResult itself - but its an interface, not a real class, so I ended up creating a method.
its still used only internally, so its "_"-prefixed

other option is make RelayingResult a class, and put this into its "toString"